src/tcgwars/logic/impl/gen4/SupremeVictors.groovy

```diff
@@ -2697,107 +3308,130 @@ public enum SupremeVictors implements LogicCardInfo {
             }
           }
           move "Encore", {
-            text "Choose 1 of the Defending Pokémon’s attacks. That Pokémon can use only that attack during your opponent’s next turn."
+            text "Choose 1 of the Defending Pokémon's attacks. That Pokémon can use only that attack during your opponent's next turn."
             energyCost C
-            attackRequirement {}
+            attackRequirement {
+              assert !defending.topPokemonCard.moves.isEmpty() : "$defending has no attacks"
+            }
             onAttack {
-              damage 0
+              def move=choose(defending.topPokemonCard.moves, defending.topPokemonCard.moves.collect({it.name}), "Encore") as Move
+              def pcs = defending
+              bc "$pcs can only use ${move.name} next turn."
+              delayed{
```

Comment:
I wonder if a getter here would be more user-friendly, but may take some time to work out, so I don't expect it in this PR.

Reply:
I like that idea a lot.